### PR TITLE
Fix mermaid render breaks with theme setting

### LINKED ISSUE #184: Mermaid.js preview breaks
It fails Mermaid graph preview with updating Mermaid.js by `:PrevimUpdateAssets`.

```bash
% git diff --name-only | xargs md5sum
2a66af0aa884e09f41086cf620b4186f *preview/_/js/lib/highlight.pack.js
07d4a62ba2e5b0e44077bd9b481fd75d *preview/_/js/lib/mermaid.min.js
```

MermaidRenderingFailure

----

Rolling back the `mermaid.min.js` to the version in this repository, it works.

Browser did not raise errors.
So Mermaid.js might have changed the way to start graph rendering.

Mermaid.js latest ver. is v9.3.0 now.

Below is the `mermaid.min.js` that fails rendering.

* [`mermaid.min.js`](https://github.com/previm/previm/files/10242922/mermaid.min.js.txt)

### PR DESCRIPTION
**Pull request by KSR-Yasuda**
Fix #184.

### CODE REVIEW

File: autoload/previm/assets.vim [hunk 1]
```diff
@@ -122,6 +122,7 @@ let s:source_map = [
 \        'path': '_/js/lib/mermaid.min.js',
 \        'url': 'https://cdn.jsdelivr.net/npm/mermaid@latest/dist/mermaid.min.js',
 \        'code': [
+\          'mermaid.initialize({startOnLoad: false});',
 \          'mermaid.init();',
```
Comment (lambdalisue) [inline autoload/previm/assets.vim:126]: :question:

Calling `mermaid.init()` immediately after `mermaid.initialize({startOnLoad: false})` doesn't make sense to me.

I'm not sure but it seems using `mermaid.init()` itself is already deprecated?
https://github.com/mermaidjs/mermaidjs.github.io/blob/master/usage.md#using-the-mermaidinit-call

**Review by lambdalisue — COMMENTED** [review]
@mattn What do you think?

File: autoload/previm/assets.vim (hunk 1 above)
Comment (KSR-Yasuda) [reply autoload/previm/assets.vim:126]: Indeed.
But without either of these two lines, Mermaid rendering fails...

Perhaps I misunderstand the way to initialization.
If you know the correct way, please fix it...

Comment (KSR-Yasuda) [reply autoload/previm/assets.vim:126]: Now I found: it should be `mermaid.run()`
(cf. <https://mermaid.js.org/config/usage.html#using-mermaid-run>).

Comment (KSR-Yasuda) [reply autoload/previm/assets.vim]: Although, `mermaid.run()` requires mermaid.js v10, at least.

Updated mermaid.js to the latest v10.7.0 together.

**Comment by KSR-Yasuda** [comment]
@lambdalisue Rebased.

**Comment by lambdalisue** [comment]
Merged. Thanks for your contribution 🎉

**Pull request merged**